### PR TITLE
MAD tank balance changes/overhall

### PR DESCRIPTION
**Pull request by Ethal-A**
the Mad tank has been changed so that it is more useful as a support infantry unit that still causes devastation
video: https://vid.me/CXgI

### CODE REVIEW

**Comment by cjshmyr** [comment]
I think I'd be willing to accept that with a few tweaks:
- Less crater intervals, so they don't appear so thick by the end in the full circle.
- Not sure about the damage yet, haven't tested it.

File: mods/ra/weapons/other.yaml [hunk 1]
```diff
@@ -263,10 +263,69 @@ Mandible:
 MADTankThump:
 	InvalidTargets: MADTank, Infantry
 	Warhead@1Dam: HealthPercentageDamage
+		Spread: 1c0
+		damage: 2
```
Comment (GraionDilach) [inline mods/ra/weapons/other.yaml]: Damage is with a capital D.

File: mods/ra/weapons/other.yaml [hunk 2]
```diff
@@ -263,10 +263,69 @@ Mandible:
...
+			Delay: 8
```
Comment (GraionDilach) [inline mods/ra/weapons/other.yaml]: The indentation is wrong here - notsure if this'll even work with that.

**Comment by Truth99A** [comment]
lets forget this happened ok?
(i have no idea how to use github lmao and in my attempts to remove the commits I messed up a tone of things to the point of no return and may need to completely delete my repository)
as for now go on the other one pull request with the exact same name

File: mods/ra/weapons/other.yaml (hunk 2 above)
Comment (Phrohdoh) [inline mods/ra/weapons/other.yaml]: It will not work with incorrect indentation.

**Pull request closed without merging**